<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rememberer</class>
 <widget class="QMainWindow" name="Rememberer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="make_deck_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать колоду</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decks_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Колоды</string>
    </property>
   </widget>
   <widget class="QLabel" name="decks_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите колоду:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_of_decks">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>950</width>
      <height>740</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="make_card_btn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>820</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Создать карточку</string>
    </property>
   </widget>
   <widget class="QLabel" name="question_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваш вопрос:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="answer_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Ваш ответ:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="question_text">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>460</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="answer_text">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>460</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="name_of_deck_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название колоды:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_of_deck_text">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>780</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="learn_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Учить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_qpic_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить картинку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_apic_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить картинку</string>
    </property>
   </widget>
   <widget class="QComboBox" name="decks_box">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stats_btn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="question_browser">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>460</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer_browser">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>460</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="show_answer_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать ответ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go_btn">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>820</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>легко</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repeat_btn">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>820</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>повторить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hard_btn">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>820</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>тяжело</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normal_btn">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>820</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>нормально</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
